File: 01_Language/01_Functions/php/ctype_digit.php
<?php

$strings = array('1820.20', '10002', 'wsl!12');
foreach ($strings as $testcase) {
    if (ctype_digit($testcase)) {
        echo "The string $testcase consists of all digits.\n";
    } else {
        echo "The string $testcase does not consist of all digits.\n";
    }
}

File: 01_Language/01_Functions/python/ctype_digit.py
# coding: utf-8

strings = ['1820.20', '10002', 'wsl!12']
for s in strings:
    if s.isdigit():
        print("The string " + s + " consists of all digits.")
    else:
        print("The string " + s + " does not consist of all digits.")
